run_md.py

```diff
@@ -0,0 +1,879 @@
+                    var_eq_dirs.append(res)
+
+            var_md_dirs = []
+            # os.path.dirname(var_lig)
+            for res in calc_dask(run_simulation, var_eq_dirs, dask_client, project_dir=project_dir):
+                if res:
+                    var_md_dirs.append(res)
+
+        finally:
+            dask_client.shutdown()
+
+        deffnm = 'md_out'
+        logging.info(f'Simulation of {var_md_dirs} were successfully finished\n')
+
+    else: # continue prev md
+        dask_client = init_dask_cluster(hostfile=hostfile, n_tasks_per_node=1, ncpu=ncpu)
+        try:
+            var_md_dirs = []
+            deffnm = f'{deffnm_prev}_{mdtime_ns}'
+            # os.path.dirname(var_lig)
+            for res in calc_dask(continue_md_from_dir, wdir_to_continue_list, dask_client,
+                                 tpr=tpr_prev, cpt=cpt_prev, xtc=xtc_prev,
+                                 deffnm_prev=deffnm_prev, deffnm_next=deffnm, mdtime_ns=mdtime_ns, project_dir=project_dir):
+                if res:
+                    var_md_dirs.append(res)
+
+        finally:
+            dask_client.shutdown()
+        logging.info(f'Continue of simulation of {var_md_dirs} were successfully finished\n')
+
+    # Part 3. MD Analysis. Run on each cpu
+    dask_client = init_dask_cluster(hostfile=hostfile, n_tasks_per_node=ncpu, ncpu=ncpu)
+    try:
+        var_md_analysis_dirs = []
+        # os.path.dirname(var_lig)
+        for res in calc_dask(run_md_analysis, var_md_dirs,
+                             dask_client, deffnm=deffnm, mdtime_ns=mdtime_ns, project_dir=project_dir):
+            if res:
+                var_md_analysis_dirs.append(res)
+    finally:
+        dask_client.shutdown()
+
+    logging.info(f'\nAnalysis of md simulation of {var_md_analysis_dirs} were successfully finished\n')
+
+    if not not_clean_log_files:
+        if wdir_to_continue_list is None:
+            for f in glob(os.path.join(wdir_md, '*', '#*#')):
+                os.remove(f)
+        else:
+            for wdir_md in wdir_to_continue_list:
+                for f in glob(os.path.join(wdir_md, '#*#')):
+                    os.remove(f)
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='''Run or continue MD simulation.\n
+    Allowed systems: Protein in water, Protein-Ligand-multiple Coenzymes, Protein-Ligand, Protein-multiple Coenzymes''')
+    parser.add_argument('-p', '--protein', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file of protein. Supported formats: *.pdb or gro')
+    parser.add_argument('-d', '--wdir', metavar='WDIR', default=None, type=filepath_type,
+                        help='Working directory. If not set the current directory will be used.')
+    parser.add_argument('-l', '--ligand', metavar='FILENAME', required=False, type=filepath_type,
+                        help='input file with compound. Supported formats: *.mol or sdf or gro')
```

Comment:
Multiple gro files are not allowed. Is it a frequent use case when a user wants to use multiple gro files as input?

Reply:
Yes, no gro files at all can be set for ligand or cofactor (in the current version). I have fixed it already in readme, but have not in the code

Reply:
OK, I agree, this will simplify the pipeline